README.md

```diff
@@ -1,2 +1,106 @@
+
+## Installing Visual Studio Code
+
+Next, in order to read and write the code, you're going to need a text editor. There are a lot of good options, but the one that we've been using is [Visual Studio Code][], particularly because of its [Live Share][Live share] feature.
+
+![Visual Studio Code][i:VS Code]
+
+## Installing `git`
+
+`git` is a version-control system: it keeps track of all of the changes made to the project, and lets several people work on the same code in parallel.
+
+### On Windows
+
+If you're on Windows, you can go [here][Install git for Windows] to download `git`.
+
+When the installer asks you to choose the default editor, you may want to select Visual Studio Code. Other than that, the default options should be just fine.
+
+![The git installer program on Windows, asking what the default editor should be, with "Visual Studio Code" selected][i:Windows, installing git]
+
+### On a Mac
+
+If you're on a Mac, you can get `git` as part of the Xcode command-line developer tools. Open up Terminal and type `git --version`:
+
+<img width="638" alt="Terminal with the command &quot;git --version&quot; typed" src="https://user-images.githubusercontent.com/56209343/89746736-f0de7000-da80-11ea-9ab5-cf48acd6b5f7.png">
+
+You'll get a pop-up asking you to install the developer tools:
+
+<img width="573" alt="The pop-up window asking if you'd like to install the tools" src="https://user-images.githubusercontent.com/56209343/89746735-f0de7000-da80-11ea-9075-abcefdc23124.png">
+
+If you don't get a pop-up, but instead get a version number like this, that means `git` is already installed.
+
+<img width="682" alt="Terminal showing the command &quot;git --version&quot; and the output &quot;git version 2.24.3 (Apple Git-128)&quot;" src="https://user-images.githubusercontent.com/56209343/89746734-f045d980-da80-11ea-8944-f2b0de3b2ecb.png">
+
+## Installing GitKraken
+
+`git` by itself is very spare, so we've been using it through a program called [GitKraken][]. (This program has a nice chart showing all of the revisions over time.) You can use GitKraken for free, but if you're a Morris student, you have access to the pro version through GitHub's student developer pack.
+
+![The screen GitKraken shows when it first launches][i:GitKraken splash page]
+
+Once you've installed GitKraken, you're going to want to download the Minnesota Pollinators code onto your computer. To do this, click the "Clone a repo" button on the left side of GitKraken, and type in the `git` URL for the project.
+
+(You can find the `git` URL of any repository by going to that repository's page on GitHub and looking under the "Code" dropdown. Here's the `git` URL for Buzz About:)
+
+![The GitHub page for Buzz About, with the "Code" dropdown menu open, showing the title "Clone with HTTPS" and the start of the URL][i:Cloning from GitHUb]
+
+Now, you should be able to see the whole history of the project!
```

Comment:
```suggestion
Now, you should be able to see the whole history of the project! Generally speaking, you will want to create a branch for any feature or change you wish to contribute and make a pull request in GitHub when you think it is ready for review to be added to the main branch in the repository. You can check out any branch locally using GitKraken by double clicking it. 
```

Reply:
Ooh, thank you, I'll add that! (I think I'll make it its own paragraph, if that's all right :slightly_smiling_face:)